monkestation/code/datums/status_effects.dm

```diff
@@ -0,0 +1,30 @@
+
+/datum/status_effect/kleptomania
```

Comment:
Couldn't seem to get it to ever activate when standing around as a monke with zero added traits.
Surrounded by people with pockets full of loot.